app/views/projects/samples/metadata/_form.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= form_with(model: @sample, url: namespace_project_sample_metadata_field_path(sample_id: @sample.id), method: :post) do |form| %>
+
+  <div id='test' class="grid gap-2" data-controller="projects--samples--metadata--add">
+    <div data-projects--samples--metadata--add-target="metadataToAdd"></div>
+    <div id='input-field-0' class='inputField'>
+      <div id="field-label-0" class="text-slate-700 dark:text-slate-400 text-sm mb-2">
+        <%= t(".field")%> 1:
+      </div>
+
+      <div class="flex space-x-2 mb-2">
+        <div class="relative z-0 w-full group">
+          <input type="text" name="key-0" id="key-0" class="block py-2.5 px-0 w-full text-sm text-gray-900 bg-transparent border-0 border-b-2 border-gray-300 appearance-none dark:text-white dark:border-gray-600 dark:focus:border-primary-500 focus:outline-none focus:ring-0 focus:border-primary-500 peer" placeholder=" " required />
```

Comment:
**nitpick:** Please update all the `gray` to `slate`

Reply:
Fixed in 24e373a7374b8f112cc0f9b44d43fcfbf1baad80